src/components/Badge/Badge.vue

```diff
@@ -25,9 +23,9 @@ export default {
   props: {
     variant: {
       type: String,
```

Comment:
I believe since the variant prop is not required, 'default' does not need to be declared as a variant
So default can be the absence of a variant, through a computed value 'noVariant' or 'defaultVariant' or something similar, where variant is null 
did you try something like that? what do you think?

Reply:
I like `default: 'default',` better, I mean it's built for cases exactly like this one (absence of a value being passed).

Reply:
sure - it doesn't hurt I guess but in this case maybe we should match the template classes to it?